tests/e2e/utils/file-reset.ts

```diff
@@ -45,11 +16,13 @@ export const resetFiles = async () => {
       stdio: 'inherit',
       cwd: process.env.TEST_APP_PATH,
     });
-  }
 
-  // wait for server to restart after modifying files
-  console.log('Waiting for Strapi to restart...');
-  // TODO: this is both a waste of time and flaky. We need to find a way to access playwright server output and watch for the "up" log to appear
-  await delay(3); // give it time to detect file changes and begin its restart
-  await pollHealthCheck(); // give it time to come back up
+    // wait for the server to restart after modifying files
+    console.log('Waiting for Strapi to restart...');
+
+    // TODO:  This is both a waste of time and flaky.
+    //        We need to find a way to access playwright server output and watch for the "up" log to appear
+    await delay(3); // give it time to detect file changes and begin its restart
+    await pollHealthCheck(); // give it time to come back up
+  }
```

Comment:
Since we're fixing this, can we just do:

```
if (!process.env.TEST_APP_PATH) {
    console.warn("Could not reset files because TEST_APP_PATH not set");
    return;
}
```